### ISSUE TITLE
FILE

### ISSUE THREAD

**Issue by n10c32y99u6** [n10c32y99u6/0522html#2]
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    
	
    <title>Auto Hacking Tool</title>
    
    
    
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    
    
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
	
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap-theme.min.css">
	
	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>

  </head>

  <body>
   
	<nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
		<div class="container-fluid">
		
			<div class="navbar-header">
				<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
					<span class="sr-only">Toggle navigation</span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				</button>
      <a class="navbar-brand" href="#">Auto Haking Tool</a>
    </div>

    
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav">
        <li class="active"><a href="HOME.html">開始 <span class="sr-only">(current)</span></a></li>
        <li><a href="FILE.html">檔案及設定</a></li>
		<li><a href="LOG.html">歷史紀錄</a></li> 
      </ul> 
      <ul class="nav navbar-nav navbar-right">
        <li><a href="#">Hacker</a></li>
		
      </ul>
    </div>
  </div>
</nav>
	
	 
	<div class="container theme-showcase" role="main">
		<div class="top"> 
			<h1>上傳</h1>
			<h2>使用者可以選擇要測試預設範本或自訂測試檔案</h2>
		</div>
		<div class="btn-group" role="group" aria-label="...">
			<a href="FILE.html"  class="btn btn-default" role="button">範例</a>
			<input id="input-1" type="file"  class="btn btn-default">
		</div>
		<div>
		 
		 </div>
	</div> 
	
	
	<div class="container theme-showcase">
		<div class="row">
			<div class="col-xs-6 col-md-3">
				<a href="#" class="thumbnail">
					<img data-src="holder.js/100%x180" alt="預覽圖片">
				</a>
			</div>
			
		</div>
		
		<div class="form-group">
			<label for="portId">Set port  (default is  9) :</label>
			<input type="password" class="form-control" id="portId">
		</div>
		
		
		<div class="btn-group">
			<button type="button" class="btn btn-danger">Penetration Test</button>
			<button type="button" class="btn btn-danger dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
				<span class="caret"></span>
				<span class="sr-only">Toggle Dropdown</span>
			</button>
			<ul class="dropdown-menu" role="menu">
				<li><a href="#">Default</a></li>
				<li><a href="#">Script 0</a></li>
				<li><a href="#">Script 1</a></li>
				<li class="divider"></li>
				<li><a href="#">Details</a></li>
			</ul>
		</div>
		
	</div>
	
	
	
	
	
 
	 
        
            
           
        
	
	
    
             
        </div>
	
	 
  
  </body>
  
  

  
</html>
